sopel/cli/run.py

```diff
@@ -0,0 +1,619 @@
+    parser_start.add_argument(
+        '--quiet',
+        action="store_true",
+        dest="quiet",
+        help="Suppress all output")
+    utils.add_common_arguments(parser_start)
+
+    # manage `configure` sub-command
+    parser_configure = subparsers.add_parser(
+        'configure', help='Sopel\'s Wizard tool')
+    parser_configure.add_argument(
+        '--modules',
+        action='store_true',
+        default=False,
+        dest='modules')
+    utils.add_common_arguments(parser_configure)
+
+    # manage `stop` sub-command
+    parser_stop = subparsers.add_parser(
+        'stop',
+        description='Stop a running Sopel instance',
+        help='Stop a running Sopel instance')
+    parser_stop.add_argument(
+        '-k', '--kill',
+        action='store_true',
+        default=False,
+        help='Kill Sopel without a graceful quit')
+    utils.add_common_arguments(parser_stop)
+
+    # manage `restart` sub-command
+    parser_restart = subparsers.add_parser(
+        'restart',
+        description='Restart a running Sopel instance',
+        help='Restart a running Sopel instance')
+    utils.add_common_arguments(parser_restart)
+
+    return parser
+
+
+def check_not_root():
+    """Check if root is running the bot.
+
+    It raises a ``RuntimeError`` if the user has root privileges on Linux or
+    if it is the ``Administrator`` account on Windows.
+    """
+    opersystem = platform.system()
+    if opersystem in ["Linux", "Darwin"]:
+        # Linux/Mac
+        if os.getuid() == 0 or os.geteuid() == 0:
+            raise RuntimeError('Error: Do not run Sopel with root privileges.')
+    elif opersystem in ["Windows"]:
+        # Windows
+        if os.environ.get("USERNAME") == "Administrator":
+            raise RuntimeError('Error: Do not run Sopel as Administrator.')
+    else:
+        stderr("Warning: %s is an uncommon operating system platform. Sopel should still work, "
+               "but please contact Sopel's developers if you experience issues." % opersystem)
+
+
+def print_version():
+    """Print Python version and Sopel version on stdout."""
+    py_ver = '%s.%s.%s' % (sys.version_info.major,
+                           sys.version_info.minor,
+                           sys.version_info.micro)
+    print('Sopel %s (running on python %s)' % (__version__, py_ver))
+    print('https://sopel.chat/')
+
+
+def print_config():
+    """Print list of available configurations from default homedir."""
+    configs = utils.enumerate_configs(DEFAULT_HOMEDIR)
+    print('Config files in %s:' % DEFAULT_HOMEDIR)
+    config = None
+    for config in configs:
+        print('\t%s' % config)
+    if not config:
+        print('\tNone found')
+
+    print('-------------------------')
+
+
+def get_configuration(options):
+    """Get an instance of configuration from options.
+
+    This may raise a ``sopel.config.ConfigurationError`` if the file is an
+    invalid configuration file.
```

Comment:
"if the file is an invalid configuration file" is a bit redundant.

How about: "if the configuration file is invalid"?

No suggested change, because GitHub won't allow selecting multiple lines to edit. :disappointed: